src/main.cpp

```diff
@@ -28,6 +30,7 @@ void print_usage(const std::string &application) {
     std::cout << "  -n          Enable natural sort order if sort order is a string representation. Default is disabled." << std::endl;
     std::cout << "  -s <...>    Sort by property; 'size', 'name', 'atime', 'mtime', 'ctime'. Default is 'size'." << std::endl;
     std::cout << "  -t <ms>     File/directory parse timeout given in milliseconds. Default is infinite (-1)." << std::endl;
+    std::cout << "  --tsep <c>  Add thousands seperator. Default is none." << std::endl;
```

Comment:
Long arguments (double dash) in key-value form are supposed to separate key and value by '=': "--tsep <c>" would be "--tsep=<c>". Then you can use "arg.value" instead of "arg.next->value".